### PR TITLE
Adding argument in Gateway Vault Authorization command

### PR DESCRIPTION
**Pull request by magesanjay**
## Scope of this PR

- __Technical__: Adding error message mapper argument in Vault Authorization command.

## Purpose of this PR

When this pull request is merged, it will add error message mapper argument in Vault Authorization command.

## Affected URLs

- https://devdocs.magento.com/guides/v2.2/payments-integrations/vault/vault-di.html#add-a-gateway-command-for-vault
- https://devdocs.magento.com/guides/v2.3/payments-integrations/vault/vault-di.html#add-a-gateway-command-for-vault

## Links to source code

Codebase Urls:

- https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Braintree/etc/di.xml#L274
- https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Braintree/etc/di.xml#L274

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/payments-integrations/vault/vault-di.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ Example: a gateway command for the authorize action is added for the Braintree v
         <argument name="client" xsi:type="object">Magento\Braintree\Gateway\Http\Client\TransactionSale</argument>
         <argument name="handler" xsi:type="object">BraintreeVaultResponseHandler</argument>
         <argument name="validator" xsi:type="object">Magento\Braintree\Gateway\Validator\ResponseValidator</argument>
+        <argument name="errorMessageMapper" xsi:type="object">Magento\Braintree\Gateway\ErrorMapper\VirtualErrorMessageMapper</argument>
```
Comment (ronak2ram) [inline guides/v2.1/payments-integrations/vault/vault-di.md]: Hi @magesanjay 
I don't find an extra argument of Vault Authorization command in [V2.1](https://github.com/magento/magento2/blob/2.1/app/code/Magento/Braintree/etc/di.xml#L248).
Can you please revert change of v2.1 doc files. we can accept these changes for the doc v2.2 and v2.3

Comment (magesanjay) [reply guides/v2.1/payments-integrations/vault/vault-di.md]: Hello @ronak2ram 
I have reverted a change in v2.1 and applying it to V2.2. and V2.3.
Thanks

Comment (ronak2ram) [reply guides/v2.1/payments-integrations/vault/vault-di.md]: thanks, @sanjay for the quick update.

**Comment by jeff-matthews** [comment]
I ran a diff on the 2.1 and new 2.2 file that you added @magesanjay,  and the only difference I see is the following on line 66:

```
<argument name="errorMessageMapper" xsi:type="object">Magento\Braintree\Gateway\ErrorMapper\VirtualErrorMessageMapper</argument>
```

Is that all that changed or am I missing something?

**Comment by magesanjay** [comment]
Hello @jeff-matthews 
Yes that is only change.

**Comment by dobooth** [comment]
running tests

**Pull request merged**